features/tests/e2e/helpers/wallets-helpers.js

```diff
@@ -119,9 +122,11 @@ const createWalletsAsync = async (table, context) => {
   }, table);
   // Add or set the wallets for this scenario
   if (context.wallets != null) {
-    context.wallets.push(...result.value);
+    context.wallets.push(...result.value.storeWallets);
+    context.mnemonics.push(...result.value.mnemonics);
   } else {
-    context.wallets = result.value;
+    context.wallets = result.value.storeWallets;
+    context.mnemonics = result.value.mnemonics;
```

Comment:
@daniloprates this one breaks E2E tests:
1. No recent transactions
2. Addresses ordering
3. Hide/show used addresses

with error message: `TypeError: Cannot read property 'push' of undefined`